lib/scout_apm/layer_converters/database_converter.rb

```diff
@@ -52,23 +51,11 @@ def skip_layer?(layer)
       DEFAULT_OPERATION = "other"
 
       def model_name(layer)
-        if layer.name.respond_to?(:model)
-          layer.name.model || DEFAULT_MODEL
-        else
-          DEFAULT_MODEL
-        end
-      rescue
-        DEFAULT_MODEL
+        layer.name.to_s.split("/").first || DEFAULT_MODEL
```

Comment:
`model` is nil for layers that don't have a name and just SQL. This instead takes the output of `to_s` to generate the final model name.

Reply:
This is better than the cascading if/else/rescue that was there before, but still seems roundabout to do a full `.to_s` only to then split it back out.  I'll think about how that could be better.